src/templates/Home/Home.module.scss

```diff
@@ -1,41 +1,56 @@
-.container {
-  display: grid;
-  height: 100%;
-  width: 100%;
-}
+@use '../../styles/variables';
 
-.underConstruction {
-  align-self: center;
-  color: #005f82;
-  display: grid;
-  gap: var(--mm-spacing);
-  grid-template-areas: 'icon' 'text';
-  grid-template-columns: 1fr;
-  grid-template-rows: max-content;
-  justify-self: center;
+.layout [class^='Layout-module--content'] {
```

Comment:
At some point we should re-visit the use of `class^=`.  Maybe we could add a `className` prop for children; although this wouldn't help with grand-children and their descendants.